### PR TITLE
Add cop to check for focused specs

### PR DESCRIPTION
**Pull request by renanborgescampos**
Detect offenses whenever focus: true arg is found

### CODE REVIEW

**Comment by andyw8** [comment]
Might also want to the non-hash form, e.g. 'describe "...", :focus`, for when `treat_symbols_as_metadata_keys_with_true_values` is enabled. (It's on by default from v3).

File: lib/rubocop-rspec.rb [hunk 1]
```diff
@@ -19,3 +19,4 @@
 require 'rubocop/cop/rspec/multiple_describes'
 require 'rubocop/cop/rspec/not_to_not'
 require 'rubocop/cop/rspec/verified_doubles'
+require 'rubocop/cop/rspec/focus'
```
Comment (andyw8) [inline lib/rubocop-rspec.rb]: It looks like this list should be in alphabetical order.

**Comment by olliebennett** [comment]
Nice initiative @renanborgescampos - your work has been taken and extended by Jared in his PR. I vote to merge #76 :)

**Comment by renanborgescampos** [comment]
Closing this PR. I vote to merge #76

**Pull request closed without merging**